ui/app/templates/components/get-credentials-card.hbs

```diff
@@ -6,24 +6,32 @@
     <p class="is-label search-label">{{@searchLabel}}</p>
   </div>
   <p class="sub-text">{{@subText}}</p>
-  <SearchSelect
-    @id={{this.id}}
-    @models={{@models}}
-    @selectLimit="1"
-    @backend={{@backend}}
-    @fallbackComponent="input-search"
-    @shouldUseFallback={{@shouldUseFallback}}
-    @onChange={{action "handleRoleInput"}}
-    @inputValue={{get this.model this.valuePath}}
```

Comment:
I think `@inputValue` was copied over accidentally as it's unnecessary here. It's used for forms to pre-fill selected items, which happens on edit